Jenkinsfile

```diff
@@ -118,8 +121,17 @@ pipeline {
                     }
                     post { always { deleteDirWin() } }
                 }
-                stage('Unit') {
-                    agent any
+                stage('Linux Unit') {
+                    agent { label 'linux' }
+                    steps {
+                        unstash 'StanSetup'
+                        setupCXX(false)
```

Comment:
By default the linux nodes have clang as their compiler I believe; we'll need to set it specifically here.

Reply:
I think you have the correct line commented out here, possibly? Or is this still waiting for GCC fixes? If so can you put the issue # in a comment directly into the Jenkinsfile for us?

Reply:
If I switch it to gcc it will not build anymore.